tests/disk_management/__init__.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+
+VHD_DIR: str = f'{os.path.dirname(os.path.abspath(__file__))}/virtual_hard_disk'
+
+
+def create_disk_test_case(testcase_klass, scenario_data: dict = None):
```

Comment:
`klass`? Is that a German library? :^)

Reply:
> class is a reserved keyword in some languages such as Java and thus cannot be used as the variable name. Using clazz or klass is a way to workaround that.

Maybe it's a bad habit. Do you want me to avoid doing that in python?

Reply:
I would understand if the name was only `class`, but here it's `testcase_klass`, so i'm not sure it's necessary

Reply:
It isn't necessary in python, ok i fix it now

Reply:
("type", "kind" and "category" are synonymes that pop up in my head, I don't know if they're usable in this context though)